### DISCUSSION TITLE
Running in a Windows VM

### DISCUSSION THREAD

**Discussion by akiross** [slint-ui/slint#1369] [Q&A]
Hello,

I tried to use a Windows VM to build and test an application that on Linux runs fine, but I'm getting an error related to OpenGL context that cannot be created (which is basically the same error reported here https://github.com/slint-ui/slint/issues/171#issuecomment-1031477333).

I understand a Win VM might be an unsupported use case; if not, are there known issues, fixes or workarounds related to this platform?

Thanks

**Comment by tronical** [comment]
I’m developing Slint on Windows in a VM, using VMware fusion.

I think what is required is either an installation of Qt (for software rendering) or an OpenGL driver that’s not the “Windows” one. For example VMware fusion installs one that works, but I suspect that Mesa might also work. It would be good if we had documentation that explained how to set up the Mesa software GL implementation for that purpose (if it works).
